src/runmode-dpdk.c

```diff
@@ -695,6 +706,13 @@ static int ConfigLoad(DPDKIfaceConfig *iconf, const char *iface)
     if (retval < 0)
         SCReturnInt(retval);
 
+    retval = ConfGetChildValueBoolWithDefault(
```

Comment:
I feel this statement is unnecessarily complex, can you break it up for readability?

Reply:
Done